liquibase-core/src/main/java/liquibase/change/core/CreateProcedureChange.java

```diff
@@ -303,7 +305,12 @@ public SqlStatement[] generateStatements(Database database) {
                 try {
                     InputStream stream = openSqlStream();
                     if (stream == null) {
-                        throw new IOException(FileUtil.getFileNotFoundMessage(path));
+                        if (ChangeLogParserConfiguration.WARN_ON_MISSING_CHANGELOGS.getCurrentValue()) {
```

Comment:
I think it would be good to extract this piece of code into a function of AbstractChange as both (CreateProcedure and CreateView extend of it), as something like this:
```
public ByteArrayInputStream returnEmptyResourceIfMissingFile(String path) {
    if (ChangeLogParserConfiguration.WARN_ON_MISSING_CHANGELOGS.getCurrentValue()) {
        Scope.getCurrentScope().getLog(getClass()).warning(FileUtil.getFileNotFoundMessage(path));
        return new ByteArrayInputStream(FileUtil.EMPTY_FILE.getBytes(StandardCharsets.UTF_8));
    }
    else {
        throw new IOException(FileUtil.getFileNotFoundMessage(path));
    }
}
```
So then if we have it available there if any other object needs to execute the same logic,  or if they need to override it to apply any other variation of it.